bitrise.yml

```
@@ -889,7 +889,7 @@ workflows:
             set -e
             set -x
 
-            # Import only the shipping locales (from shipping_locales.txt) onRelease
+            # Import  the locales from firefoxios-l10n
```

```suggestion
            # Import the locales from firefoxios-l10n
```

nit, pending the rest of the conversation